docs/stats/index.rst

```diff
@@ -77,8 +80,42 @@ with the Li \& Ma formula:
    >>> significance_on_off(n_on=18, n_off=97, alpha=1. / 10, method='lima')
    2.2421704424844875
 
+Confidence Intervals
+--------------------
+
+Assume you measured 6 counts in a Poissonian counting experiment with an
+expected background :math:`b = 3`. Here's how you compute the 90% upper limit
+on the signal strength :math:`\\mu`:
```

Comment:
You are mistaken about what gammapy.stats.fc_find_acceptance_region_poisson does. For a given background and mu it returns the acceptance interval (x_min, x_max) such that the integrated probability is the confidence level.

The case covered here is you measured a certain x and now you are looking for the corresponding confidence interval (mu_1, mu_2).

There is a difference between acceptance and confidence interval. Normally the user is interested in the confidence interval.